backend/team/models.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+
+
+class TechStack(models.Model):
+    #Model
+    tech_name = models.CharField(max_length=60)
+
+    class Meta:
+        verbose_name = 'Tech Stack'
+        verbose_name_plural = 'Tech Stack'
+
+    def __str__(self):
+        return self.tech_name
+
+
+class Member(models.Model):
```

Comment:
here the member model is equivalent to userprofile 
there should be a model named team having
year
captain:: foreign to the user
vice-captain:: m2m user
core team:: m2m user

Reply:
@harsh-ux add the more fields to the same member model as u were supposed to  for userprofile here